packages/disploy/src/structs/ChatInputInteractionResolvedOptions.ts

```diff
@@ -0,0 +1,54 @@
+import type { Snowflake } from 'discord-api-types/v10';
+import { Base } from './Base';
+import type { BaseChannel } from './BaseChannel';
+import type { ChatInputInteraction } from './ChatInputInteraction';
+import { PartialChannel } from './PartialChannel';
+import { PartialGuildMember } from './PartialGuildMember';
+import { User } from './User';
+
+export class ChatInputInteractionResolvedOptions extends Base {
+	public interaction: ChatInputInteraction;
+	public members: Map<Snowflake, PartialGuildMember>;
+	public users: Map<Snowflake, User>;
+	public channels: Map<Snowflake, BaseChannel>;
+	constructor(interaction: ChatInputInteraction) {
+		super(interaction.app);
+		/**
+		 * The interaction the resolved options belong to.
+		 */
+		this.interaction = interaction;
+
+		/**
+		 * The resolved members.
+		 */
+		this.members = new Map();
+		/**
+		 * The resolved users.
+		 */
+		this.users = new Map();
+		/**
+		 * The resolved channels.
+		 */
+		this.channels = new Map();
```

Comment:
JSDocs live above the declaration, not the definition.

```suggestion
	/**
	* The interaction the resolved options belong to.
	*/
	public interaction: ChatInputInteraction;
	
	/**
	* The resolved members.
	*/
	public members: Map<Snowflake, PartialGuildMember>;
	
	/**
	* The resolved users.
	*/
	public users: Map<Snowflake, User>;
	
	/**
	* The resolved channels.
	*/
	public channels: Map<Snowflake, BaseChannel>;
	
	constructor(interaction: ChatInputInteraction) {
		super(interaction.app);
		this.interaction = interaction;
		this.members = new Map();
		this.users = new Map();
		this.channels = new Map();
```